packages/teams-js/test/public/dialog.spec.ts

```diff
@@ -220,4 +221,45 @@ describe('Dialog', () => {
       expect(submitMessage.args).toEqual(['someResult', ['someAppId']]);
     });
   });
+  describe('sendMessageToParentFromDialog', () => {
```

Comment:
can we verify that sendMessageToParent was called correctly using unit tests?

Reply:
It would be nice to talk about it. Are we worrying because these calls would be generated from a different instance of appSDK

Reply:
It looks like you have tests for this on 239, maybe I missed them last time